storm-core/src/jvm/backtype/storm/scheduler/resource/RAS_Node.java

```diff
@@ -44,74 +43,76 @@
  */
 public class RAS_Node {
     private static final Logger LOG = LoggerFactory.getLogger(RAS_Node.class);
-    private Map<String, Set<WorkerSlot>> _topIdToUsedSlots = new HashMap<String, Set<WorkerSlot>>();
-    private Set<WorkerSlot> _freeSlots = new HashSet<WorkerSlot>();
-    private final String _nodeId;
-    private String _hostname;
-    private boolean _isAlive;
-    private SupervisorDetails _sup;
-    private Double _availMemory;
-    private Double _availCPU;
-    private List<WorkerSlot> _slots;
-    private List<ExecutorDetails> _execs;
-    private Map<WorkerSlot, List<ExecutorDetails>> _slotToExecs;
+    private Map<String, Set<WorkerSlot>> topIdToUsedSlots = new HashMap<String, Set<WorkerSlot>>();
+    private Set<WorkerSlot> freeSlots = new HashSet<WorkerSlot>();
+    private final String nodeId;
+    private String hostname;
+    private boolean isAlive;
+    private SupervisorDetails sup;
+    private Double availMemory;
+    private Double availCPU;
+    private Cluster cluster;
+    private Topologies topologies;
```

Comment:
There are a lot of changes to this class that merely change existing variable names, like `_nodeId` -> `nodeId`. Just need to make sure new code uses a consistent notation with what is already in the class.